Apps/BabylonScripts/ValidateFiles.ps1

```diff
@@ -0,0 +1,32 @@
+Push-Location $PSScriptRoot
+Write-Host "Installing npm packages for comparison..."
+npm install

```

Comment:
Is there any reason we shouldn't just make people npm install and stop checking the files in directly?

Reply:
that would require all developers to start using npm. Right now in this current form npm is only used for validation and isn't required for development if you aren't changing these script files